FEXCore/Source/Interface/Core/Frontend.cpp

```diff
@@ -1092,10 +1094,12 @@ const uint8_t* Decoder::AdjustAddrForSpecialRegion(const uint8_t* _InstStream, u
   return _InstStream - EntryPoint + RIP;
 }
 
-void Decoder::DecodeInstructionsAtEntry(const uint8_t* _InstStream, uint64_t PC, uint64_t MaxInst,
-                                        std::function<void(uint64_t BlockEntry, uint64_t Start, uint64_t Length)> AddContainedCodePage) {
+void Decoder::DecodeInstructionsAtEntry(
+  const uint8_t* _InstStream, uint64_t PC, uint64_t MaxInst,
+  std::function<void(fextl::set<uint64_t> BlockEntryPoints, uint64_t Start, uint64_t Length)> AddContainedCodePage) {
```

Comment:
`BlockEntryPoints` should be a reference parameter, shouldn't it? I'm mildly surprised this compiles, since the lambda you pass for `AddContainedCodePage` below does use a reference.